tests/staggered_dslash_test_utils.h

```diff
@@ -503,7 +503,7 @@ struct StaggeredDslashTestWrapper {
       if (std::isnan(spinor_out_norm2)) { failed = true; }
 
       double cuda_spinor_out_norm2 = blas::norm2(*cudaSpinorOut);
-      printfQuda("Results: CPU=%f, CUDA=%f, CPU-CUDA=%f\n", spinor_ref_norm2, cuda_spinor_out_norm2, spinor_out_norm2);
+      printfQuda("Results: CPU=%f, CUDA=%f, (CPU-CUDA)/CPU=%e\n", spinor_ref_norm2, cuda_spinor_out_norm2, 1.0-cuda_spinor_out_norm2/spinor_ref_norm2);
```

Comment:
@jcosborn can you apply this change to dslash_test_utils.h.  Despite the fact it arose due to a misunderstanding, it's a good change to have.  Though I think it probably would be better to have `1.0 - sqrt(....)`